src/test/java/io/stargate/sgv2/jsonapi/api/v1/FindAndUpdateIntegrationTest.java

```diff
@@ -1347,6 +1413,8 @@ private void insertDoc(String docJson) {
         .when()
         .post(CollectionResource.BASE_PATH, keyspaceId.asInternal(), collectionName)
         .then()
+        // Sanity check: let's look for non-empty inserted id
```

Comment:
Needed because we always get 200 response; failure indicated differently so need to get inserted-id.
(not checking masked a few failures within invalid json being sent)